intranet/templates/polls/poll.html

```diff
@@ -40,10 +40,15 @@ <h3>
         for
         {% for group in poll.groups.all %}
             {{ group }}
-            {% if not forloop.last %},{% endif %}
+            {% if not forloop.last %},
+            {% else %}
+            {% endif %}
         {% empty %}
-            everyone
+            everyone 
         {% endfor %}
+        {% if is_polls_admin and not poll.before_start_time %}
+            &bull; {{ poll.get_voted_string }}
```

Comment:
I haven't tested the UI. Is the bullet here really necessary?

Reply:
Yes, it keep the format consistent because the previous two pieces of info on that line are separated by a bullet.

Reply:
Ah. Makes sense.